impeller/typographer/backends/skia/typographer_context_skia.cc

```diff
@@ -580,6 +583,10 @@ std::shared_ptr<GlyphAtlas> TypographerContextSkia::CreateGlyphAtlas(
     height_adjustment = 0;
     atlas_context->UpdateRectPacker(nullptr);
     atlas_context->UpdateGlyphAtlas(new_atlas, {0, 0}, 0);
+  } else {
+    // If any part of the old atlas is reused the atlas generation can be
+    // treated the same as existing glyphs will not be moved.
+    new_atlas->SetAtlasGeneration(last_atlas->GetAtlasGeneration());
```

Comment:
I don't think this is necessary is it?  new_atlas points to last_atlas in this case.

Reply:
ahh good catch.